rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -325,6 +330,13 @@ def decode_transaction(
                 write_cursor=write_cursor,
                 history=events,
             )
+            if all([event.balance.amount == ZERO and event.asset != A_ETH for event in events]) is True:  # noqa: E501
```

Comment:
I would not do it like this. This is a hack and does not fit the decoding process. Specifically becuase it adds another implicit post-decoding rule, which takes linearly more time, the more events a transaction has.

What you need to do is:
1. If a **token** transfer is 0 value, not decode it at all, in the token transfers decoding.
2. Then here check that the number of events of is empty. If yes ignore the transaction, saying it's most probably a phishing transaction with the link you already have as comment.